worker/worker.go

```diff
@@ -110,6 +110,7 @@ func toHostInteraction(m metrics.Metric) (hostdb.Interaction, bool) {
 type Bus interface {
 	RecordHostInteraction(hostKey consensus.PublicKey, hi hostdb.Interaction) error
 	ContractsForSlab(shards []object.Sector) ([]bus.Contract, error)
+	Health() (bus.HealthResponse, error)
```

Comment:
why would the worker need this?

Reply:
There is really no other working endpoint that I could use in an integration test to verify that a worker is successfully connected to a bus. Same for the health endpoint on the bus. Nothing tells us that the submodules on the backend are working.

That's why I added the health endpoints. That and because it is useful anyway since you can consume the endpoint from other tools to monitor the health of your setup.

Reply:
Do we *need* to verify that they're connected? Why not attempt to do the thing you need to do, and if you aren't connected, you'll get an error? After all, a successful `Ping` doesn't mean that the next call can't fail!

Putting a `Ping` method on a type like `SingleAddressWallet` illustrates how strained this is. YAGNI.